### ISSUE TITLE
.deb install produces error

### ISSUE THREAD

**Issue by johnwbaxter** [frenkel/hfl#56]
I'm trying to install 0.3.9 using the .deb onto Ubuntu (pop_os) 22.04 and the install fails.

I get:
```
Installation Failed

Child process exited with code 100
```

I get more logging using dpkg to install it.

```
 timer-for-harvest depends on libssl1.1; however:
  Package libssl1.1 is not installed.
```

If I try to install that lib, I get:
```

Package libssl1.1 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
```

And that's it really!

Thanks!

**Comment by frenkel** [comment]
That's unfortunate. Most distributions still provide an old libssl for compatibility which allows me to provide one binary for old and new distributions. I think this will need to be changed then. I'll have a look within one week and will let you know.

In the meantime you could try to clone this repo and build it yourself. There is a big chance that just works. Basic instructions can be found in https://github.com/frenkel/timer-for-harvest#building

**Comment by johnwbaxter** [comment]
Ok, what anyone else will need to do is this to make it work on 22.04:

```
echo "deb http://security.ubuntu.com/ubuntu impish-security main" | sudo tee /etc/apt/sources.list.d/impish-security.list

sudo apt-get update
sudo apt-get install libssl1.1
```

All good now!

**Comment by frenkel** [comment]
Thanks for the workaround! I'll see how simple it is to provide a correct .deb and otherwise I'll add your instructions to the README.

**Comment by johnwbaxter** [comment]
No worries, thanks for the app!! :)

**Comment by lcherpit** [comment]
> Ok, what anyone else will need to do is this to make it work on 22.04:
> 
> ```
> echo "deb http://security.ubuntu.com/ubuntu impish-security main" | sudo tee /etc/apt/sources.list.d/impish-security.list
> 
> sudo apt-get update
> sudo apt-get install libssl1.1
> ```
> 
> All good now!

Unfortunately, it's not working anymore. ubuntu impish release is no longer maintained and the repo source is gone. libssl1 can be found here : http://security.ubuntu.com/ubuntu/pool/main/o/openssl/?C=M;O=D

**Comment by frenkel** [comment]
Sorry about that, haven't had time to build a seperate .deb for Ubuntu 22.04 yet. Will do so soon!

**Comment by lcherpit** [comment]
It's not a big deal to install that lib in the meantime. Once installed it works fine. Thank you for your work!

**Comment by frenkel** [comment]
I've just release new binaries that work on Ubuntu 22.04!
https://github.com/frenkel/timer-for-harvest/releases/tag/v0.3.10

**Comment by artmatsak** [comment]
@frenkel Thank you for addressing this issue! However, the latest version doesn't start on Ubuntu 20.04 now, produces this error:

```
timer-for-harvest: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
```

**Comment by frenkel** [comment]
Yeah, sorry about that. The thing is can't currently get it to build on older versions even though I didn't change my code. If anybody knows what's going wrong let me know:
https://github.com/frenkel/timer-for-harvest/runs/7575688198?check_suite_focus=true

For now please just use 0.3.9 because there is no actual difference in user functionality between them.

**Comment by frenkel** [comment]
Fixed! The problems were caused by a shared cache between the two different build environments. I've published v0.3.11 with .deb files for both old and new Ubuntu.
